docs/connect.html

```diff
@@ -98,6 +98,10 @@
     <li><code>GET /connectors/{name}/status</code> - get current status of the connector, including if it is running, failed, paused, etc., which worker it is assigned to, error information if it has failed, and the state of all its tasks</li>
     <li><code>GET /connectors/{name}/tasks</code> - get a list of tasks currently running for a connector</li>
     <li><code>GET /connectors/{name}/tasks/{taskid}/status</code> - get current status of the task, including if it is running, failed, paused, etc., which worker it is assigned to, and error information if it has failed</li>
+    <li><code>PUT /connectors/{name}/pause</code> - pause the connector and its tasks, which stops message processing until the connector is resumed</li>
+    <li><code>PUT /connectors/{name}/resume</code> - resume a paused connector (or do nothing if the connector is not paused)</li>
+    <li><code>POST /connectors/{name}/restart</code> - restart a connector task</li>
```

Comment:
@hachikuji Yeah, to be honest, we have much better structure [in the Confluent docs](http://docs.confluent.io/2.0.1/connect/userguide.html#rest-interface). Part of the reason that didn't make it in here is because it's currently pretty painful to format all that cleanly (and requires a lot of time to port given the way we've documented it for Confluent). Hopefully this will get easier once we port the docs to an easier to manage format.

It's certainly not impossible with the raw HTML version, but we should really have body & response formats including field names & descriptions, error codes, examples, etc... If you want to add some of that, I'm open to it.